agent/acs/handler/attach_eni_handler_common.go

```diff
@@ -15,113 +15,71 @@ package handler
 
 import (
 	"fmt"
-	"time"
 
 	"github.com/aws/amazon-ecs-agent/ecs-agent/logger"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/logger/field"
 
-	apieni "github.com/aws/amazon-ecs-agent/agent/api/eni"
 	"github.com/aws/amazon-ecs-agent/agent/data"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
 	"github.com/aws/amazon-ecs-agent/agent/utils"
 	"github.com/aws/amazon-ecs-agent/agent/wsclient"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/acs/model/ecsacs"
+	apieni "github.com/aws/amazon-ecs-agent/ecs-agent/api/eni"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/utils/arn"
 	"github.com/aws/aws-sdk-go/aws"
 
 	"github.com/cihub/seelog"
 	"github.com/pkg/errors"
 )
 
 // ackTimeoutHandler remove ENI attachment from agent state after the ENI ack timeout
```

Comment:
nit. The comment on line 35 needs to be updated as well.

Reply:
Good catch! I will update this comment to reflect the updated name of the struct.